packages/block-library/src/embed/edit.js

```diff
@@ -277,6 +278,14 @@ const EmbedEdit = ( props ) => {
 					label={ label }
 					insertBlocksAfter={ insertBlocksAfter }
 				/>
+				<Caption
```

Comment:
Just a small suggestion 🙇 
As per the other caption text labels, the below label makes much more sense 
`label={ __( 'Embed caption text' ) }`

Reply:
Thanks For that @Rishit30G 🚀, I missed it.